R/system.R

```diff
@@ -234,6 +234,15 @@ glue_exploratory <- function(text, .transformer, .envir = parent.frame()) {
   ret
 }
 
+# Wrapper around glue::glue() to resolve our ${} notation used for Salesforce Filter.
```

Comment:
Add comment that this expects @ to be already resolved.

Reply:
Added comment.